service/src/main/java/uk/gov/hmcts/reform/fpl/service/StandardDirectionOrderGenerationService.java

```diff
@@ -161,12 +97,6 @@ private DocmosisRespondent buildRespondent(RespondentParty respondent) {
         ).orElse(ImmutableList.of());
     }
 
-    public String getApplicantName(Applicant applicant) {
-        return ofNullable(applicant.getParty())
-            .map(ApplicantParty::getOrganisationName)
-            .orElse("");
-    }
-
     private String formatTitle(int index, Direction direction, List<DirectionConfiguration> directionConfigurations) {
 
         // default values here cover edge case where direction title is not found in configuration. Reusable for CMO?
```

Comment:
For this I would say yes, CMO doesn't have a directionConfig so we could create an overloaded method that just takes in index and direction and then calls this method which also passes an empty list